### ISSUE TITLE
Warn about Kobweb / Compose version mismatch

### ISSUE THREAD

**Issue by bitspittle** [varabyte/kobweb-intellij-plugin#9]
Can we discover those versions from the project's Gradle model? Worst case, we can parse `libs.versions.toml` directly, but that's more fragile. What if a user isn't using version catalogs?

**Comment by Nxllpointer** [comment]
Yes this can be done using the gradle tooling api. Probably by either making the kobweb gradle plugin build a model, or what i have seen the kotlin gradle plugin doing, injecting custom code when running gradle from the tooling api

**Comment by Nxllpointer** [comment]
Actually i think just implementing this in the gradle plugin and logging a warning is the proper solution

**Comment by bitspittle** [comment]
Not a bad thought! Two possible problems I see with the Gradle approach:

1. If your Compose and Kobweb versions are mismatched, you might generate some wild errors that may dominate the warning/error output list, so the user never even sees the warning from Gradle
1. People might ignore it because they aren't sure how to fix it

If we use the IDE to detect this, we can show a message via a notification bubble that's 1) really easy to see and 2) includes a link that the user clicks on which could just fix this for them.

BTW, it's possible we should do both things. The Gradle plugin should list a warning, AND the IDE should have integrations for detecting and helping with it.

**Comment by Nxllpointer** [comment]
Fair. We can log a warning and add some version mismatch info to the Gradle model. Will probably start working on the model tomorrow
